libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -5728,6 +5731,17 @@ bool GCS_MAVLINK::try_send_message(const enum ap_message id)
         send_scaled_pressure3();
         break;
 
+    case MSG_AIRSPEED:
+#if AP_AIRSPEED_ENABLED
+        {
+            AP_Airspeed *airspeed = AP_Airspeed::get_singleton();
+            if (airspeed != nullptr) {
+                airspeed->send_mavlink_airspeed(chan);
+            }
+        }
+#endif
+        break;
```

Comment:
```suggestion
        break;
#endif
```

not doing this means we lose a sanity check